routes/courses.js

```diff
@@ -1,96 +1,123 @@
+// courses.js
+
+// Specify strict mode for 'secure' code
 'use_strict';
 
+// Import dependencies and initilize global constants
+
```

Comment:
importing stuff at the top is pretty standard & people can look up what packages do. lets remove comments about packages